prime-router/src/main/kotlin/fhirengine/azure/FHIRProcessFunctions.kt

```diff
@@ -55,4 +66,49 @@ class FHIRProcessFunctions(
         actionHistory.trackLogs(actionLogger.logs)
         workflowEngine.recordAction(actionHistory)
     }
+
+    @FunctionName("convert")
```

Comment:
Remove.  We are only adding features as we get use cases/requirements.  This may be added back in Q3.